resilience4j-ratpack/src/main/java/io/github/resilience4j/ratpack/circuitbreaker/monitoring/endpoint/CircuitBreakerChain.java

```diff
@@ -108,10 +107,13 @@ public void execute(Chain chain) throws Exception {
                     CircuitBreakerEventsEndpointResponse response = new CircuitBreakerEventsEndpointResponse(
                         eventConsumerRegistry
                             .getAllEventConsumer()
-                            .flatMap(CircularEventConsumer::getBufferedEvents)
+                            .stream()
+                            .flatMap(circuitBreakerEventCircularEventConsumer -> circuitBreakerEventCircularEventConsumer
```

Comment:
Added a method to return stream of `BufferedEvents` and reused it. Couldn't create similar method to return `flux` since `reactor` dependency is not present in the project.